src/traverse/__tests__/node-path-test.js

```diff
@@ -281,6 +281,44 @@ describe('NodePath', () => {
     );
 
     expect(aCharPath.getNextSibling()).toBe(bCharPath);
+
+  });
+
+  it('getParent/getChild', () => {
+    const ast = parser.parse('/a(bc)d/');
+
+    //console.log(JSON.stringify(ast, null, '  '));
+
+    bodyPath = NodePath.getForNode(ast.body);
```

Comment:
Use `const`, or `let` (if a var is supposed to be changed), when define variables. Assigning without const/let creates global variable (basically it escapes to global, and pollutes it).

```js
const bodyPath = ...
```

Ditto for other local vars created.